src/ftc.ts

```diff
@@ -0,0 +1,50 @@
+const completionSpec: Fig.Spec = {
+  name: "ftc",
+  description: "",
```

Comment:
Here a description should be provided:
```suggestion
  description: "",
```